button/generic-data-pipeline
# Two initial views to be used for daily chartio dashboards.

These 2 initial views will be very helpful with efficiently updating the underlying data that supports chartio views. Currently, if anything needs to be changed about those queries (e.g. approval method, currency change) it involves editing every single query/chart. Using these type of views will limit the places we need to make changes. These views can be edited & improved along the way as well.

Testing entails:
1. Bring up docker mysql.
2. Create chartio database.
3. Manually create currency table by getting the create statement from prod chartio.
4. Run db-migrate locally.

Additionally, the views are there in chartio already actually, under experiment db so you can run queries against the view to see them live.

@LiweiButton for review, cc: @seanjsummers and @jiaqi216 as FYIs.
